pkg/handler/handler.go

```diff
@@ -20,6 +20,8 @@ import (
 	authorizationv1 "k8s.io/api/authorization/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/kubernetes/pkg/apis/core"
```

Comment:
This is wrong, it needs to be
```suggestion
	corev1 "k8s.io/api/core/v1"
```